chain/postgresdb/pgdb/pgdb.go

```diff
@@ -147,22 +158,12 @@ func (p *Store) Last(ctx context.Context) (*chain.Beacon, error) {
 	LIMIT 1`
```

Comment:
Any reason not to use `FETCH FIRST` instead?

Reply:
I don't have a strong preference about using `FETCH FIRST` vs the current `LIMIT 1` approach.
The reasons we went with the current direction:
- The current approach is simpler and thus more readable and approachable.
- It's also more portable, so we could adapt this into a "generic" SQL package in the future, supporting multiple DB storage.

Reply:
That being said, we can switch to `FETCH FIRST`, if you'd prefer using PG cursors instead.

Reply:
My understanding was the it was the converse, no? `FETCH` is generic SQL whereas `LIMIT` is not, but let's keep LIMIT for now. 👍🏻